Assignment4/Anddrex-140032/task06.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1oRyMbw92qd2yr8hofEAO1vjQxBVNJeKu
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials"
+
+"""Leemos el fichero RDF de la forma que lo hemos venido haciendo"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example5.rdf", format="xml")
+
+"""Create a new class named Researcher"""
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**
+
+"""
+
+# TO DO
+# Visualize the results
+ns = Namespace("http://somewhere#")
+g.add((ns.University, RDF.type, RDFS.Class))
+for s, p, o in g.triples((ns.University,None,None)):
+  print(s,p,o)
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+
+# TO DO
+# Visualize the results
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+for s, p, o in g.triples((None,RDFS.subClassOf,None)):
+  print(s,p,o)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+
+# TO DO
+# Visualize the results
+ns = Namespace("http://somewhere#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.Researcher, vcard.FN, Literal("Jane Smith")))
+for s, p, o in g.triples((None,None,Literal("Jane Smith"))):
+  print(s,p,o)
+#for s, p, o in g:
+#  print(s,p,o)
+
+"""**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"""
+
+# TO DO
+# Visualize the results
+ns = Namespace("http://somewhere#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith,vcard.FN,Literal("Jane Smith")))
+g.add((ns.JaneSmith,vcard.Given,Literal("Jane")))
+g.add((ns.JaneSmith,vcard.Family,Literal("Smith")))
+for s, p, o in g.triples((ns.JaneSmith,None,None)):
+  print(s,p,o)
+#Estaba probando si al meterle diferentes nombre a propiedades en vcard que ocurría en las triples
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+# Visualize the results
+ns = Namespace("http://somewhere#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((vcard.works,RDF.type,RDF.Property))
+g.add((ns.UPM,RDF.type,ns.University))
+g.add((ns.JhonSmith,vcard.works,ns.UPM))
```

Comment:
have you checked if vcard.works is an existing property in vcard? I think it does not exist.